ui/src/notebooks/components/panel/NotebookPanel.tsx

```diff
@@ -92,7 +92,7 @@ const NotebookPanel: FC<Props> = ({index, children, controls}) => {
 
   useEffect(() => {
     updateMeta(index, {panelRef} as PipeMeta)
-  })
+  }, [])
```

Comment:
I believe having this run only once on mount breaks the jump to feature. I fiddled with not having to hit `updateMeta` so much but couldn't get the feature to work otherwise

Reply:
aight. i'll jump back on that then. i was debugging when something updates the meta, and saw an explosion! maybe the idea is to limit rendering in the first place.... i'll figure it out

Reply:
offlined a test condition: `when a pipe is added and moved in order, the jump to feature should scroll to the notebook entry exposed in the minimap`
tested that ^^ and it seems to work with the change